daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -1,21 +1,30 @@
 ---
 type: docs
-title: "Metadata API reference"
+title: "Sidecar Metadata API reference"
 linkTitle: "Metadata API"
 description: "Detailed documentation on the Metadata API"
 weight: 800
 ---
 
-Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present). Each loaded components also has information about supported features in the form of component's capabilites.
 
-The Dapr metadata API also allows you to store additional information in the format of key-value pairs.
+These features are only available for State store, Pub/Sub and Binding components. For example, below table shows building blocks and their capabilites. This list might grow in future and only represents the current supported their capabilites of loaded components - 
+
+Building Block | Supported capabilites
+---------------| ------------
+State Store    | Etag, Transcation, Actor, Query API
+Pub/Sub        | TTL
+Binding        | INPUT_BINDING, OUTPUT_BINDING
 
-Note: The Dapr metatada endpoint is for instance being used by the Dapr CLI when running dapr in standalone mode to store the PID of the process hosting the sidecar and the command used to run the application.
+The Dapr metadata API also allows you to store additional information in the format of key-value pairs.
```

Comment:
```suggestion
## Labels

The metadata API allows you to store additional label information in the format of key-value pairs. These are ephemeral in-memory and are not persisted if a sidecar is reloaded. This information should be added at the time of a sidecar creation after the application has started. 
```

Reply:
Can you check that label information (as I have put here) is in fact in-memory only? Is this true statement above ^^^

Reply:
yes, it is in-memory only